tests/src/test_projects/reentrancy/mod.rs

```diff
@@ -0,0 +1,116 @@
+use fuels::{prelude::*, tx::ContractId};
+
+abigen!(
+    AttackerContract,
+    "test_artifacts/reentrancy_attacker_contract/out/debug/reentrancy_attacker_contract-abi.json",
+);
+
+abigen!(
+    TargetContract,
+    "test_artifacts/reentrancy_target_contract/out/debug/reentrancy_target_contract-abi.json",
+);
+
+#[tokio::test]
```

Comment:
I would at least separate these into successful tests and those that revert modules plus another one to contain the setup. This makes it easier to navigate because chunks of code are contained together

Reply:
What would be a good structure for this, something like the following?

- `tests/src/test_projects/reentrancy/mod.rs`
- `tests/src/test_projects/reentrancy/pass.rs`
- `tests/src/test_projects/reentrancy/fail.rs`